frontend/package.json

```diff
@@ -3,10 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
+    "@testing-library/jest-dom": "^5.11.4",
```

Comment:
I had some issues with the intellisense of jest with the default version of this lib. Fix upgrade to the last one.